grails-app/conf/UrlMappings.groovy

```diff
@@ -88,6 +88,26 @@ class UrlMappings {
             action = [DELETE: "removeItem"]
         }
 
+        "/api/stockMovementItems/getStockMovementItems/$id"(parseRequest: true) {
```

Comment:
Assuming ID is the requisition ID, we probably want the REST endpoint to look like this

```
/api/stockMovements/$id/stockMovementItems
/api/stockMovements/$id/editPageItems
/api/stockMovements/$id/pickPageItems
/api/stockMovement/$id/packPageItems
```
These requests should still be delegated to the StockMovementItemApi, but I think it'll be good to use REST conventions here.

To get a single stock movement id from within a parent you could map the URL as
/api/stockMovements/$parentId/stockMovementItems/$childId

Or use the simpler:
```
/api/stockMovementItems/$id
```

For the child resource you could probably map these URLs
```
/api/stockMovements/$parentId/editPageItems/$childId
/api/stockMovements/$parentId/pickPageItems/$childId
/api/stockMovement/$parentId/packPageItems/$childId
```

Or again just expose the stockMovementItem and all of its subtypes
```
/api/editPageItems/$id
/api/pickPageItems/$id
/api/packPageItems/$id
```

The advantage of the child resource URLs is that the associated endpoint could validate that stock movement item with ID ${childId} was actually contained within the parent stock movement with ID ${parentId}. Whereas the `/api/stockMovementItems/$id` does allow for that. 

I don't think we really have a "contains" validation requirement right now so let's stick with the simple endpoints.